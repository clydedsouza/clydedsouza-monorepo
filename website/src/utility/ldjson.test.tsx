import { render } from "@testing-library/react";
import { JSONLD } from "./ldjson";

describe("LD-JSON", () => {
  it("should render default LD-JSON information", () => {
    const { container } = render(<>{JSON.stringify(JSONLD)}</>);
    expect(container).toMatchSnapshot();
  });
});
