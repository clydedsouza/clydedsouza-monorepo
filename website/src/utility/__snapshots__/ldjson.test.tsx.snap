// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LD-JSON should render default LD-JSON information 1`] = `
<div>
  {"@context":"http://schema.org","@type":"Person","givenName":"Clyde D'Souza","gender":"Male","email":"clyde@clydedsouza.net","image":"https://files.clydedsouza.net/site/clydedsouza.jpg","alumniOf":"AUT University","worksFor":"Xero","jobTitle":"Software Engineer","birthPlace":"Mumbai, India","contactPoint":{"@type":"PostalAddress","name":"Clyde D'Souza","email":"clyde@clydedsouza.net","url":"https://clydedsouza.net"},"name":"Clyde D'Souza","url":"https://clydedsouza.net","sameAs":["https://linkedin.com/in/clydedz/","https://twitter.com/clydedz","https://github.com/ClydeDz/","https://www.youtube.com/@clydedz","https://www.skillshare.com/r/user/clydedsouza","https://stackoverflow.com/users/11480716/clyde-dsouza","https://www.goodreads.com/clydedz","https://ko-fi.com/clydedsouza","https://www.twitch.tv/clydedsouzatv","https://www.facebook.com/ClydeDz/","https://blog.clydedsouza.net"],"address":{"@type":"PostalAddress","addressLocality":"Auckland CBD","addressRegion":"Auckland","addressCountry":"New Zealand"},"subjectOf":[{"@type":"CreativeWork","about":"Mama, Tell Me a Story: A Collection of Short Bedtime Stories","thumbnailUrl":"https://files.clydedsouza.net/portfolio/mama-tell-me-a-story.png","sameAs":["https://www.amazon.com/dp/B0819H55PD/ref=cm_sw_su_dp","https://play.google.com/store/books/details?id=M229DwAAQBAJ","https://magicblox.com/books/2699-mama-tell-me-a-story-a-collection-of-short-bedtime-stories#.X46IEtAzbtR","https://www.kobo.com/us/en/ebook/mama-tell-me-a-story","https://leanpub.com/mama-tell-me-a-story","https://www.barnesandnoble.com/w/mama-tell-me-a-story-clyde-dsouza/1135265538?ean=2940163107247","https://gumroad.com/l/mtmas","https://www.goodreads.com/book/show/48756914-mama-tell-me-a-story","https://www.instagram.com/mamatellmeastorybook/"]},{"@type":"CreativeWork","about":"AI, Tell Me a Story: A Collection of Short Bedtime Stories","thumbnailUrl":"https://files.clydedsouza.net/portfolio/aitellmeastory.png","sameAs":["https://www.amazon.com/AI-Tell-Me-Story-Collection-ebook/dp/B0C3CYTWCG/","https://www.goodreads.com/book/show/139478926-ai-tell-me-a-story","https://www.instagram.com/aitellmeastory/"]}]}
</div>
`;
